src/Samples/CSharp/Analyzers/CSharpAnalyzers/CSharpAnalyzers.Vsix/project.json

```diff
@@ -0,0 +1,12 @@
+﻿{
+  "dependencies": {
+    "Microsoft.VisualStudio.Settings.14.0": "14.3.25407",
+    "Microsoft.VSSDK.BuildTools": "14.3.25407"
```

Comment:
Not clear to me why this is necessary. If we were building on Dev15 I would have imagined the Dev15 tools would have gotten picked up. Was that not the case?